test/Interop/Cxx/enum/c-enums-NS_OPTIONS-swift-named-options.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %empty-directory(%T/cache)
+// RUN: %target-swift-frontend %s -I %S/Inputs -typecheck -module-cache-path %T/cache -enable-experimental-cxx-interop 2>&1 | %FileCheck --allow-empty %s
+
+// REQUIRES: objc_interop
+
+import CenumsNSOptions
+
+// CHECK-NOT: warning: imported declaration 'OptionOne' could not be mapped to 'SwiftOptionOne'
```

Comment:
This is the motivation for the patch, we are seeing many such warnings for any NS_SWIFT_NAME'd member inside an NS_OPTION.